src/utils.ts

```diff
@@ -293,14 +294,47 @@ export function getEnsTextRecord(ens: string, record: string, network = '1') {
   return call(provider, ENS_RESOLVER_ABI, [address, 'text', [ensHash, record]]);
 }
 
-export async function getSpaceUri(id, network = '1') {
+export async function getSpaceUri(
+  id: string,
+  network = '1'
+): Promise<string | null> {
   try {
     return await getEnsTextRecord(id, 'snapshot', network);
   } catch (e) {
-    return false;
+    console.log(e);
+    return null;
   }
 }
 
+export async function getEnsOwner(ens: string): Promise<string | null> {
+  const registryAddress = '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e';
+  const provider = getProvider('1');
+  const ensRegistry = new Contract(
+    registryAddress,
+    'function owner(bytes32) view returns (address)',
```

Comment:
It throws an error here 
```suggestion
   ['function owner(bytes32) view returns (address)'],
```